From: Sebastian Ramacher <sramacher@debian.org>
Date: Thu, 14 Jul 2016 23:01:01 +0200
Subject: Use local intersphinx mappings

---
 doc/en/conf.py | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/doc/en/conf.py b/doc/en/conf.py
index c631484..3eec549 100644
--- a/doc/en/conf.py
+++ b/doc/en/conf.py
@@ -348,6 +348,22 @@ intersphinx_mapping = {
     "pluggy": ("https://pluggy.readthedocs.io/en/latest", None),
     "python": ("https://docs.python.org/3", None),
 }
+def check_object_path(key, url, path):
+    if os.path.isfile(path):
+        return {key: (url, path)}
+    return {}
+
+intersphinx_mapping = {}
+intersphinx_mapping.update(
+    check_object_path('python',
+                      'http://docs.python.org/',
+                      '/usr/share/doc/python%d.%d/html/objects.inv' % \
+                       sys.version_info[:2]))
+# The Debian packaging for pluggy does not yet generate the documentation
+intersphinx_mapping.update(
+    check_object_path('pluggy',
+                      'https://pluggy.readthedocs.io/en/latest',
+                      '/usr/share/doc/python3-pluggy/html/objects.inv'))
 
 
 def configure_logging(app: "sphinx.application.Sphinx") -> None:
